packages/app/components/drop/drop-free.tsx

```diff
@@ -69,10 +71,24 @@ const defaultValues = {
   hasAcceptedTerms: false,
   notSafeForWork: false,
 };
+
+const { useParam } = createParam<{
+  checkoutSuccess?: boolean;
+}>();
+
 export const DropFree = () => {
   const isDark = useIsDarkMode();
-  const { data: userProfile } = useMyInfo();
-  const maxEditionSize = userProfile?.data?.profile.verified ? 100000 : 50;
+  const { user: userProfile } = useUser();
+  const [checkoutSuccess] = useParam("checkoutSuccess", {
+    parse: (value) => value === "true",
+    initial: false,
+  });
+  const editionSizeCredit =
```

Comment:
Could you add some code comments what this is?

Reply:
Done